plugins/woocommerce-blocks/assets/js/blocks/single-product/save.tsx

```diff
@@ -4,7 +4,9 @@
 import { InnerBlocks, useBlockProps } from '@wordpress/block-editor';
 
 const Save = () => {
-	const blockProps = useBlockProps.save();
+	const blockProps = useBlockProps.save( {
+		className: 'woocommerce',
```

Comment:
Unlike the product page, the `single-product` block doesn't add the `woocommerce` class, which is needed to apply the missing styles in the selector.